definitions/variable/hydrogen.yaml

```
@@ -0,0 +1,16 @@
+# This yaml file adds needed variables for E3M´s model data upload
+
+- Capacity|Hydrogen|Electricity|Alkaline:
+    description: Installed (available) capacity to produce hydrogen from electrolysis using an Alkaline elecrolyser
+    unit: GW
+- Capacity|Hydrogen|Electricity|PEM:
+    description: Installed (available) capacity to produce hydrogen from electrolysis using an Proton Exchange Membrane (PEM) elecrolyser
+    unit: GW
+
+- Storage Capacity|Hydrogen Storage:
```

This should be consistent with the next variable "Storage Capacity|Hydrogen|Tank" so that the variable tree is correctly shown in the Scenario Explorer.
```suggestion
- Storage Capacity|Hydrogen:
```